oct/cli/package/common_options.py

```diff
@@ -13,22 +13,44 @@ def package_options(func):
     :param func: Click CLI command to decorate
     :return: decorated CLI command
     """
+    option(
+        '--stage',
+        '-s',
+        'upgrade_stage',
+        type=Choice(['current', 'next', 'fork', 'base']),
```

Comment:
`build` and `install`? I added only those that branch out of a single key but I can add the rest too.